Packs/MicrosoftGraphUser/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Microsoft Graph User
+- Added a support for a self deployed app flow. Please note that the ***msgraph-user-change-password*** command is only supported in this flow.
```

Comment:
the rn here are wrong
self deployed app was supported before, what you added is support for getting access on behalf of a user